src/rust/engine/process_execution/src/remote.rs

```diff
@@ -817,7 +817,14 @@ impl crate::CommandRunner for CommandRunner {
       &build_id, action_digest, cached_response_opt
     );
     if let Some(cached_response) = cached_response_opt {
+      context
+        .workunit_store
+        .increment_counter(Metric::RemoteExecutionActionCacheUncached, 1);
```

Comment:
This is the same for both the if and else block. Is that intentional? If so, I think this could be factored up to the line above.

Reply:
Typo. This one needs to be the "cached" metric. 🤦